index.html

```diff
@@ -8,34 +8,204 @@
+                        <span class="contact"><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img src="image/Icon - Menu.svg" alt="Contact Us"></a></span>
+                </nav>
+            <div class="img_background">
+                    <h1 class="header" >I’m Mariane<br> 
+                        Glad to see<br> you!</h1>
+                    <p class="content">Hello I’m a software developer! I can help you build a product , feature or website
+                         Look through some of my work and experience! If you like what you see and have a 
+                         project you need coded, don’t hestiate to contact me.</p>
+                <div class="connect"><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img src="image/Social Media.svg" alt="Let's Connect"></a></div>
+                 <div class="social">
+                    <ul>
+                        <li><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img class="twitter" src="image/twitter.svg" alt="twitter"></a></li>
+                        <li><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img class="linkedin" src="image/Linkedin icon.svg" alt="Linkedin"></a></li>
+                        <li><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img class="medium" src="image/medium.svg" alt="Medium"></a></li>
+                        <li><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img class="github" src="image/github.svg" alt="GitHub"></a></li>
+                        <li><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img class="angellist" src="image/angellist.svg" alt="Angellist"></a></li>
+                    </ul>
+                 </div>
+            </div>
+            </div>
         </section>
+
+        <section>
+            <div class="border_project2">
+                <div class="image_project2">
+                    <img src="image/Snapshoot Portfolio.svg" alt="Snapshoot Portfolio">
+                </div>
+                <h1 class="tonic">Tonic</h1>
+                <div class="grid-container">
+                    <div class="grid-item1">CANOPY</div>
+                    <div class="grid-item2"><img src="image/Counter.svg" alt="Counter"></div>
+                    <div class="grid-item3">Back End Dev</div>
+                    <div class="grid-item4"><img src="image/Counter.svg" alt="Counter"></div>
+                    <div class="grid-item5">2015</div>
```

Comment:
- [ ] I think it's better to use the unordered list `ul and li` instead of `div.` for semantic reasons and user experience (UX).